website/pages/docs/connect/terminating_gateway.mdx

```diff
@@ -0,0 +1,99 @@
+Client applications can issue plain HTTP requests even when connecting to servers that require HTTPS.
+
+## Prerequisites
+
+Each terminating gateway needs two things:
+
+1. A local Consul agent to manage its configuration.
+2. General network connectivity to services within its local Consul datacenter.
+
+Terminating gateways also require that your Consul datacenters are configured correctly:
+
+- You'll need to use Consul version 1.8.0.
+- Consul [Connect](/docs/agent/options#connect) must be enabled on the datacenter's Consul servers.
+- [gRPC](/docs/agent/options#grpc_port) must be enabled on all client agents.
+- [Centralized configuration](/docs/agent/options#enable_central_service_config) must be enabled.
+
+Currently, [Envoy](https://www.envoyproxy.io/) is the only proxy with terminating gateway capabilities in Consul.
+
+- Terminating gateway proxies receive their configuration through Consul, which
+  automatically generates it based on the gateway's registration. Currently Consul
+  can only translate terminating gateway registration information into Envoy
+  configuration, therefore the proxies acting as terminating gateways must be Envoy.
+
+Sidecar proxies that send upstream traffic through a gateway aren't
+affected when you deploy terminating gateways. If you are using non-Envoy proxies as
+Connect sidecars they will continue to work for traffic directed at services linked to
+a terminating gateway as long as they discover upstreams with the
+[/health/connect](/api/health#list-nodes-for-connect-capable-service) endpoint.
+
+## Running and Using a Terminating Gateway
+
+You must complete the following steps to configure a terminating gateway to proxy traffic from services in the Consul service mesh:
+
+1. Start an Envoy proxy using the [envoy subcommand](/docs/commands/connect/envoy#terminating-gateways), specifying the `terminating` gateway type.
+2. Create a `terminating-gateway` [configuration entry](/docs/agent/config-entries/terminating-gateway) that defines 
+a set of services that the gateway will proxy traffic to. The config entry can be applied via the 
+[CLI](/docs/commands/config/write) or [API](/api/config#apply-configuration).
+3. Ensure that [Consul intentions](/docs/commands/intention#consul-intention) are set up between the source services in the mesh 
+and the gateway's linked services. The intentions must reference the destination service's name and not the gateway itself.
+
+## Terminating Gateway Configuration
+
+Ingress gateways are defined similarly to other services registered with Consul, with two exceptions.
+The first is that the [kind](/api/agent/service#kind) must be "ingress-gateway". Second,
+the ingress gateway service definition may contain a `Proxy.Config` entry just like a
+Connect proxy service, to define opaque configuration parameters useful for the actual proxy software.
+
+Terminating gateways are defined similarly to other services registered with Consul, with two exceptions.
```

Comment:
Is this and the above paragraph duplicates?

Reply:
Woops, yes. Fixed.